backend/src/test/java/heartbeat/util/StatusChangedItemsListAndCycleTimeInfosListFixture.java

```diff
@@ -18,8 +18,8 @@ public static List<StatusChangedItem> STATUS_CHANGED_ITEMS_LIST_OF_REAL_DONE_COL
 
 	public static List<CycleTimeInfo> CYCLE_TIME_INFOS_LIST_OF_REAL_DONE_COLUMN() {
 		return List.of(CycleTimeInfo.builder().column("DONE").day(EXPECT_DAYS).build(),
-				CycleTimeInfo.builder().column("BLOCK").day(0.0).build(),
-				CycleTimeInfo.builder().column("IN PROGRESS").day(EXPECT_DAYS).build());
+				CycleTimeInfo.builder().column("IN PROGRESS").day(EXPECT_DAYS).build(),
```

Comment:
`IN PROGRESS` and others should be a constant...

Reply:
done